scripts/zones/East_Sarutabaruta/mobs/River_Crab.lua

```diff
@@ -11,4 +13,15 @@ require("scripts/globals/fieldsofvalor");
 
 function onMobDeath(mob,killer,ally)
     checkRegime(ally,mob,92,1);
+
+    mob = mob:getID();
```

Comment:
This was a copy from the spipi script.  I did notice that but forgot to update it.  Ty for catching it.